test/parallel/test_tensorflow2_keras.py

```diff
@@ -523,7 +523,7 @@ def compute_and_apply_gradients_in_tf_function(var_list, **kwargs):
         aggregation_counter = hvd_optimizer._agg_helper.counter.numpy()
         assert aggregation_counter == total_num_of_steps % backward_passes_per_step
 
-    def test_distributed_optimizer_with_local_vars(self):
+    def test_partial_distributed_optimizer(self):
```

Comment:
Should `test_distributed_optimizer_with_local_vars` be kept? The function it tests is not covered by `test_partial_distributed_optimizer`

Reply:
The test now calls `PartialDistributedOptimizer()` which in turn calls `register_local_var()` function, so it's the same.